eclass/ros-catkin.eclass

```diff
@@ -192,8 +195,15 @@ ros-catkin_src_configure_internal() {
 # @DESCRIPTION:
 # Configures a catkin-based package.
 ros-catkin_src_configure() {
-	export CATKIN_PREFIX_PATH="${EPREFIX}/usr"
-	export ROS_ROOT="${EPREFIX}/usr/share/ros"
+	if [ -f "${ROSPREFIX%/}/setup.bash" ]; then
```

Comment:
Please don't worry about this for the moment -- I'm still trying to figure things out and am trying to find a better way to do this.